pkg/services/ngalert/state/historian/loki.go

```diff
@@ -277,11 +292,18 @@ func statesToStreams(rule history_model.RuleMeta, states []state.StateTransition
 	return result
 }
 
-func (h *RemoteLokiBackend) recordStreams(ctx context.Context, streams []stream, logger log.Logger) error {
+func (h *RemoteLokiBackend) recordStreams(ctx context.Context, streams []stream, orgID int64, logger log.Logger) error {
 	if err := h.client.push(ctx, streams); err != nil {
 		return err
 	}
+
 	logger.Debug("Done saving alert state history batch")
+	org := fmt.Sprint(orgID)
+	samples := 0
+	for _, s := range streams {
+		samples += len(s.Values)
```

Comment:
I think we should have `grafana_alerting_historian_batch_bytes_total` to get a sense of the data being sent over the wire.

You'd have to do it at a client level. My suggestion is that within `httpLokiClient.push` you use `len(enc)` or return the size from the function (having to modify the interface) so that it can be recorded at an interface level.